examples/hello-world/hello.fs

```diff
@@ -2,7 +2,14 @@ require gbhw.fs
 
 also gb-assembler
 
-$61 ==> 
+title: EXAMPLE
+
+$40 ==> reti,
```

Comment:
This interruption handlers look like a sane default. Should we do them in cartridge?  The games can override them if needed.

Reply:
Maybe a bit early to tell, the same is achieved with `di,` in a way it seems.

Reply:
I'll take a look at that a bit later, and try on the emulator if there seem to be differences.